lib/Net/Twitter/Core.pm

```diff
@@ -36,7 +36,7 @@ has clienturl       => ( isa => 'Str', is => 'ro', default => 'http://search.cpa
 has _base_url       => ( is => 'rw' ); ### keeps role composition from bitching ??
 has _json_handler   => (
     is      => 'rw',
-    default => sub { JSON->new->utf8 },
+    default => sub { JSON::MaybeXS->new->utf8 },
```

Comment:
you could actually leave this and the other invocations as ````JSON->new````, JSON::MaybeXS imports JSON as a constant, per the docs:
````my $json_obj = JSON->new; # returns a Cpanel::JSON::XS or JSON::PP object````